packages/brain/src/index.ts

```diff
@@ -88,19 +100,26 @@ await brainDb.initialize(signerApi, validatorApi);
 logger.debug(brainDb.data);
 
 // CRON
-export const cron = new Cron(
+export const reloadValidatorsCron = new CronJob(
   60 * 1000,
-  signerApi,
-  signerUrl,
-  validatorApi,
-  brainDb
+  new ReloadValidators(
```

Comment:
We saw that this was causing issues. I would export a function instead of passing here a public method of a  new object